libraries/api-extractor/config/post-copy.json

```diff
@@ -0,0 +1,12 @@
+{
+  "copyTo": {
+    "./samples/api-documenter": [ 
+      "../api-documenter/gulpfile.js",
+      "../api-documenter/package.json",
+      "../api-documenter/run.cmd",
+      "../api-documenter/tsconfig.json",
+    ],
+    "./samples/api-documenter/src": [ "../api-documenter/src/**" ]
+  },
+  "shouldFlatten": false
+}
```

Comment:
@iclanton  It does not get compiled.  The idea is that third parties can compile it themselves if they want to.

---
In reply to: [134041820](https://github.com/Microsoft/web-build-tools/pull/325#discussion_r134041820) [](ancestors = 134041820)